chainer_pytorch_migration/ignite/extensions.py

```diff
@@ -296,8 +296,13 @@ def serialize(self, serializer):
 
         # Lets save torch objects using torch interface
         if isinstance(serializer, chainer.serializer.Serializer):
-            snap_path = self._chainer_trainer.get_extension(
-                '_Snapshot').filename
+            try:
+                snap_path = self._chainer_trainer.get_extension(
+                    '_Snapshot').filename
+            except ValueError:
+                # Look for the multinode snapshot instead
+                snap_path = self._chainer_trainer.get_extension(
+                    '_MultiNodeSnapshot').snapshot.filename
```

Comment:
I think we should detect if the extension is a snapshot one when registering it instead of just using this dirty approach. Let me rewrite it!
Thanks🙇‍♂️